build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: bgppolicies.crd.antrea.io
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - nodeSelector
+                - advertisements
+                - bgpPeers
```

Comment:
I wonder if it really makes sense to have `bgpPeers` as a required property here? It feels like an empty list would be acceptable: in that case we would simply not connect to any peers.

Reply:
My bad, we should allow empty `bgpPeers`. After all, it's okay for a router to configure bgp process without adding any peer.